src/index.js

```diff
@@ -189,8 +189,8 @@ export default class Undo {
   /**
    * Sets events listeners to allow keyboard actions support.
    */
-  setEventListeners() {
-    const { holder } = this.editor.configuration;
```

Comment:
@MarioRodriguezS  As you need to accept both (strings and DOM elements) I think a good approach could be:

```javascript
const holderElem = typeof holder === 'string' ? document.getElementById(holder) : holder
```

What do you think?